Makefile

```diff
@@ -23,7 +23,8 @@ MICROSERVICES=cmd/core-metadata/core-metadata cmd/core-data/core-data \
 	cmd/security-secrets-setup/security-secrets-setup cmd/security-proxy-setup/security-proxy-setup \
 	cmd/security-secretstore-setup/security-secretstore-setup \
 	cmd/security-file-token-provider/security-file-token-provider \
-	cmd/security-bootstrap-redis/security-bootstrap-redis
+	cmd/security-bootstrap-redis/security-bootstrap-redis \
+	cmd/edgex-secrets-config/edgex-secrets-config
```

Comment:
Drop the `edgex-` prefix and just have it called `secrets-config` so naming is consistent. i.e. no other services have the prefix.